apps/judicial-system/backend/src/app/modules/file/file.service.ts

```diff
@@ -352,14 +359,27 @@ export class FileService {
         CaseFileCategory.DEFENDANT_CASE_FILE,
       ].includes(file.category)
     ) {
-      await this.messageService.sendMessagesToQueue([
-        {
+      const messages: Message[] = []
+
+      if (theCase.origin === CaseOrigin.LOKE) {
+        messages.push({
+          type: MessageType.DELIVERY_TO_POLICE_CASE_FILE,
+          user,
+          caseId: theCase.id,
+          elementId: file.id,
+        })
+      }
+
+      if (theCase.courtCaseNumber) {
```

Comment:
We should only attempt to send the files to court if a court case has been created.